library/src/pivotal-ui-react/radio/radio.js

```diff
@@ -10,16 +10,26 @@ var Radio = React.createClass({
     onChange: React.PropTypes.func,
     id: React.PropTypes.string,
     className: React.PropTypes.string,
-    style: React.PropTypes.object
+    style: React.PropTypes.object,
+    disabled: React.PropTypes.bool
   },
 
   render: function() {
-    const {className, style, children, ...others} = this.props;
+    const {className, style, children, disabled, ...others} = this.props;
     const props = mergeProps({className: className, style: style}, {className: 'radio'});
+
     return (
       <div {...props}>
-        <label>
-          <input type="radio" {...others}/>
+        <label
+          className={disabled ? 'type-neutral-5' : ''}
+          style={disabled ? {cursor: 'not-allowed'} : undefined}
```

Comment:
Taking each point in turn.

1. `not-allowed`
    When we put this into the sandbox and set the `cursor: default`, we don't get the behaviour we want. The normal cursor is displayed for the label, but the not-allowed cursor is displayed for the radio button. We feel that this is confusing, we want the cursor to be consistent.

2. Style
    We looked at adding a `disabledLabel` style, but then changed our minds. It seemed to break the general OOCSS pattern of providing general-purpose CSS elements, rather than structural elements. 

    We may have misinterpreted the pattern. We *do* however feel that *some* style should be applied. Other form elements, when disabled, show similar visual indicators.